.github/workflows/prebuild.yaml

```diff
@@ -0,0 +1,234 @@
+name: Make Prebuilds
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '.github/workflows/prebuild.yaml'
+
+# When a new Node.js version is released or for any other reason that a build
+# needs to be triggered, amend this file and force-push.
+
+env:
+  # These should generally be the same:
+  CANVAS_VERSION_TO_BUILD: "2.6.1" # no "v" prefix. TODO switch to $GITHUB_REF if we switch to `on: release`
+  CANVAS_PREBUILT_VERSION: "2.6.1"
+
+jobs:
+  Linux:
+    strategy:
+      matrix:
+        # Make sure to set this for the other builds also!
+        node: [8, 9, 10, 11, 12, 13, 14]
+    name: Node.js ${{ matrix.node }} on Linux
+    runs-on: ubuntu-latest
+    container:
+      image: chearon/canvas-prebuilt:7
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Build
+        env:
+          OS: ${{ runner.os }}
+        run: |
+          npm install -g node-gyp
+          npm install --ignore-scripts
+          . prebuild/$OS/preinstall.sh
+          cp prebuild/$OS/binding.gyp binding.gyp
+          node-gyp rebuild -j 2
+          . prebuild/$OS/bundle.sh
+
+      - name: Test binary
+        run: node -e "require('.')"
```

Comment:
IIRC I couldn't run the tests here because they weren't in the NPM tarball - that or I was just lazy :D. Might be good opportunity to do the tests instead of just testing loading. Possibly uninstall Cairo et all first to make sure the prebuilds did work.